components/Embrace.client.vue

```diff
@@ -0,0 +1,313 @@
+<template>
+  <div
+    class="w-full flex items-center justify-center bg-leaf-700 rounded-2xl px-8 py-8 text-gray-50 text-center"
+  >
+    <form
+      class="flex flex-col items-center"
+      @submit.prevent.stop="checkAndDisplayPreview"
+    >
+      <p class="text-xl md:text-2xl mb-6 font-semibold whitespace-pre-wrap">
+        {{ embracePage?.data.form_title || 'Share your thoughts' }}
+      </p>
+      <div class="grid grid-cols-2 md:grid-cols-4 gap-6 text-left">
+        <TextField
+          v-model="fullName"
+          class="col-span-2"
+          name="fullName"
+          type="text"
+          :title="embracePage?.data.full_name_label || 'Your full name'"
+          :placeholder="embracePage?.data.full_name_placeholder || 'Write your full name'"
+          :warning="fullNameWarning"
+          dark
+        />
+        <TextField
+          v-model="form.email"
+          class="col-span-2"
+          type="email"
+          name="email"
+          :title="embracePage?.data.email_label || 'Email'"
+          :placeholder="embracePage?.data.email_placeholder || 'email@example.com'"
+          :warning="warningsMap['email']"
+          dark
+        />
+        <div class="col-span-2">
```

Comment:
I think the reason they are separate might be that previous devs expected country would often already be defined/inferred from referring page.  That doesn't mean we can't have a wrapper to combine them since I think it makes sense to always show it anyway.  I'll add creating a wrapper component to the existing BankSearch sub-task rather than addressing it in this PR.

Reply:
Yeah I get that but I also see that the bank search actually requires a country so you can actually not use the bank search without a country, as much as I see it right?

Reply:
and yeah that's a good idea to add a separate sub issue for this!!

Reply:
In theory you could use the search separately if you passed the country as a parameter to the page but I think even if we had those cases it would be better to show it as pre-selected so the user could change it, 

I did notice that the country selection component is used by itself in at least one place. 

Regardless, I've added this to a new ticket :)